app/models/pickup_post.rb

```diff
@@ -0,0 +1,12 @@
+class PickupPost < ActiveRecord::Base
+  include Order
+
+  belongs_to :site
+  belongs_to :post
+
+  validates :post_id, presence: true, uniqueness: {scope: :site_id}
+
+  before_validation do
+    self.order = (self.class.maximum(:order) || 0) + 1 if !order
```

Comment:
It is very similar to Category's code :cry: 

https://github.com/bm-sms/daimon-news/pull/655/files#diff-af69b723432608c351fb1dd5db9e8348L23

How about extracting `assign_default_order`?

```ruby
module Orderable
...
  included do
    before_validation do
      assign_default_order unless order
    end
  end

  def assign_default_order
    self.order = (self.class.maximum(:order) || 0) + 1
  end
...
end

class PickupPost
  include Orderable
...
end

class Category
    before_validation do
      assign_default_order if ancestry_changed?
    end
end
```

I'm very happy if you make sure Category's behavior is not changed.
(Perhaps above code doesn't work well :sweat_smile:)